scripts/generate_seo_tags.rb

```
@@ -0,0 +1,232 @@
+        end
+      end
+    end
+    
+    # Write updated content back to file
+    File.write(file_path, doc.to_html)
+    return true
+  rescue => e
+    puts "Error updating metadata for #{file_path}: #{e.message}"
+    return false
+  end
+end
```

RuboCop (1.73)

[convention] Assignment Branch Condition size for update_html_with_metadata is too high. [<10, 32, 12> 35.61/23]

(Metrics/AbcSize)